recipes/qt.py/existance_check.py

```diff
@@ -0,0 +1,25 @@
+import site
+import sys
+import os
+
+
+def existance_check():
+    """Returns True if Qt.py is found, else returns False"""
+
+    found = False
+    site_package_locations = site.getsitepackages()
+    for location in site_package_locations:
+        if os.path.exists(os.path.join(location, 'Qt.py')):
+            print('Found Qt.py in: ' + location)
+            found = True
+    if not found:
+        print('Qt.py was not found in any of the locations: ' + ', '.join(
+            site_package_locations))
+    return found
+
+
+if __name__ == '__main__':
+    if existance_check():
+        sys.exit(0)
+    else:
+        sys.exit(1)
```

Comment:
This looks like an overly complex script for a simple test :smile: 
Are there any `import` test that could substitute this?

Reply:
No, unfortunately, there isn't. Qt.py requires you to have PySide, PyQt4, PySide2 or PyQt5 installed and will raise an error if none of them are found. I can't add any of these bindings as requirements in `meta.yaml` as I wish the users of Qt.py to decide which of these bindings they want to use.

I'm not aware of any other way to tell if the `Qt.py` file is in place. Would you recommend some other way?

Reply:
Then add `PySide` as a test requirement
```
test:
    requires:
       - PySide
    imports:
       - Qt.py
```